test/Template/PlatesTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Expressive\Template;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Zend\Expressive\Template\Plates as PlatesTemplate;
+use League\Plates\Engine;
+use Zend\Expressive\Template\TemplatePath;
+
+class PlatesTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->platesEngine = new Engine();
+    }
+
+    public function testConstructorWithEngine()
+    {
+        $template = new PlatesTemplate($this->platesEngine);
+        $this->assertTrue($template instanceof PlatesTemplate);
```

Comment:
@ezimuel I saw your comment to @samsonasik about this, but there's another way to do it:

```php
$this->assertInstanceOf(PlatesTemplate::class, $template);
```

The `::class` constant will resolve even an aliased class to its canonical form, making it useful for this.